app.py

```diff
@@ -5,26 +5,27 @@
 class DatabaseLoader:
     # main function
     def __init__(self):
-        server = environ.get("SERVER")
-        user = environ.get("USER")
-        password = environ.get("PASSWORD")
-        dbname = environ.get("DBNAME")
-        self.setup_db(server, user, password, dbname)
+        self.server = environ.get("SERVER")
+        self.user = environ.get("USER")
+        self.password = environ.get("PASSWORD")
+        self.dbname = environ.get("DBNAME")
 
     # takes the csv and inserts it into the db
-    def setup_db(self, server, user, password, dbname):
-        conn = psycopg2.connect("host=" + str(server) + " port='5432' dbname="
-                                + str(dbname) + " user=" + str(user)
-                                + " password=" + str(password))
+    def setup_db(self):
+        conn = psycopg2.connect(host=self.server,
+                                port=5432,
+                                dbname=self.dbname,
+                                user=self.user,
+                                password=self.password)
+
         cur = conn.cursor()
 
         # does table exist
         tb_exists = "select exists(" \
                     "select relname from pg_class where relname='"\
                     + "wine_reviews" + "')"
         cur.execute(tb_exists)
-        execute = cur.fetchone()[0]
-        if not execute:
+        if cur.fetchone()[0] is False:
```

Comment:
@elmiko are you sure about this one?  is fetchone()[0] going to return a boolean, or some other thing that might evaluate False (like None, empty list, etc).

Reply:
i checked the api and it looks like the tuple that is returned gives a `True/False` in that position, i wanted to avoid the same with something that would evaluate true the other way lol

Reply:
haha, okay! thx